server/push/controllers/send_push.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import config from '../../config';
+import webpush from 'web-push';
+
+/**
+ * Send push with payload to endpoint with it's authentication key.
+ * @param {string} endpoint endpoint from subscription
+ * @param {string} keys authentication key and p256dh key from subscription
+ * @param {string} payload payload data send to endpoint
+ * @return {promise} result includes statusCode, headers, body
+ */
+function sendPush(endpoint, keys, payload) {
+  return new Promise((resolve, reject) => {
+    webpush.setGCMAPIKey(config.serverInfo.pushServerKey);
+    webpush.setVapidDetails(
+      'https://github.com/romandev/absolute',
```

Comment:
I tried, but web-push throws error for here
  if (typeof subject !== 'string' || subject.length === 0) {
    throw new Error('The subject value must be a string containing a URL or ' +
      'mailto: address. ' + subject);
  }
So, use variable for the parameter to readability